supervision/utils/internal.py

```diff
@@ -1,5 +1,78 @@
 import functools
 import warnings
+from typing import Callable
+
+
+def deprecated_parameter(
+    old_parameter: str,
+    new_parameter: str,
+    map_function: Callable = lambda x: x,
+    warn_message: str = "Warning: '{old_parameter}' in '{function_name}' "
```

Comment:
Please do change `warn_message` to `warning_message`. I try to avoid using abbreviated names.